src/sqlitestorage.cpp

```diff
@@ -139,9 +139,7 @@ class mKCal::SqliteStorage::Private
 
     bool addIncidence(const Incidence::Ptr &incidence, const QString &notebookUid);
     int loadIncidences(sqlite3_stmt *stmt1,
-                       bool loadRecurringIncidences = false,
-                       int limit = -1, QDateTime *last = NULL, bool useDate = false,
-                       bool ignoreEnd = false);
+                       bool loadRecurringIncidences = false);
```

Comment:
The true case for this becomes unused for now. Was that because of used in the search method PR?

Made a comment there now whether it should be loading content or not. In the meanwhile the related code is not too much so one option could be to remove it now for completeness and add back later if there's need.

Reply:
I agree. I've removed this also and will add it again (maybe in a different form) when needed, like for the search method.